packages/paste-website/src/components/homepage/Experiment.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {useTheme} from '@twilio-paste/theme';
+import {AspectRatio} from '@twilio-paste/aspect-ratio';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
+
+const Experiment: React.FC = () => {
+  const theme = useTheme();
+  return (
+    <Box
+      // @ts-ignore
+      background={`linear-gradient(-180deg, ${theme.backgroundColors.colorBackgroundBody} 0%, ${theme.backgroundColors.colorBackgroundBrandHighlightLightest} 100%);`}
```

Comment:
Should this be a token? Or should we allow `string` on background?

Reply:
This should probably be a token.